File: jar/src/main/java/jline/api/sn/SnIsMixedModel.java
package jline.api.sn;

import static jline.api.sn.SnHasMixedClasses.snHasMixedClasses;

import jline.lang.NetworkStruct;

public final class SnIsMixedModel {
    private SnIsMixedModel() {}

    /**
     * Checks if the network is a mixed model.
     *
     * @param sn the NetworkStruct object for the queueing network model
     * @return true if the network has mixed classes, false otherwise
     */
    public static boolean snIsMixedModel(NetworkStruct sn) {
        return snHasMixedClasses(sn);
    }
}

File: jar/src/main/kotlin/jline/api/sn/SnIsMixedModel.kt
package jline.api.sn

import jline.lang.NetworkStruct

/**
 * Checks if the network is a mixed model.
 *
 * @param sn the NetworkStruct object for the queueing network model
 * @return true if the network has mixed classes, false otherwise
 */
fun snIsMixedModel(sn: NetworkStruct): Boolean {
    return snHasMixedClasses(sn)
}
/**
 * Stochastic network IsMixedModel algorithms
 */
@Suppress("unused")
class SnismixedmodelAlgo {
    companion object {
        // Class documentation marker for Dokka
    }
}